h/search/query.py

```diff
@@ -89,6 +115,16 @@ def __call__(self, search, params):
                  "unmapped_type": "boolean"}}
         )
 
+    def _date_parser(self, str_value):
+        try:
+            return float(str_value)
+        except ValueError:
+            try:
+                date = parse(str_value)
+                return (date - dt.utcfromtimestamp(0)).total_seconds() * 1000.0
```

Comment:
This code fails if the user supplies an ISO date string with a timezone specifier. This will happen in practice because the API includes timezone specifiers when it formats the created and updated fields, and the most common use case for this would be to pass the full value of created/updated from a previous API request as the "search_after" field for the next request:

```
>>> updated_date_from_prev_api_request = '2018-08-27T11:02:15.107224+00:00'
>>> parse(updated_date_from_prev_api_request) - dt.utcfromtimestamp(0)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can't subtract offset-naive and offset-aware datetimes
>>> 
```

Reply:
Looking for feedback here-how do you think it should handle this? Should it assume if a timezone isn't specified that it is local time or utc time? Do you like the existing conversion and just if it specifies the timezone, don't run the conversion?